server/tests/test_route_ticket.py

```diff
@@ -0,0 +1,158 @@
+import pytest
+from flask import Flask
+from flask_jwt_extended import create_access_token
+from models import db
+from models.user import User, Role, AccountStatus
+from models.ticket import Ticket, TicketStatus
+from models.ticket_message import TicketMessage
+from routes.ticket import tickets_bp
+
+
+@pytest.fixture
+def app():
+    """Create test Flask application"""
+    app = Flask(__name__)
+    app.config["TESTING"] = True
+    app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///:memory:"
+    app.config["JWT_SECRET_KEY"] = "test-secret-key"
+    app.config["WTF_CSRF_ENABLED"] = False
+
+    # Initialize extensions
+    db.init_app(app)
+
+    from flask_jwt_extended import JWTManager
+
+    JWTManager(app)
+
+    # Register blueprint
+    app.register_blueprint(tickets_bp, url_prefix="/api")
+
+    with app.app_context():
+        db.create_all()
+        yield app
+        db.session.remove()
+        db.drop_all()
+
+
+@pytest.fixture
+def client(app):
+    """Create test client"""
+    return app.test_client()
```

Comment:
You can use the already existing client fixture that is already in `conftest.py`.